src/app/containers/Timestamp/index.test.jsx

```diff
@@ -41,148 +90,116 @@ describe('Timestamp', () => {
-    expect(renderedWrapper[0].children[0].data).toMatch(regexDate);
-    expect(renderedWrapper[1].children[0].data).toMatch(
-      /Updated [0-9]{1,2} \w+ [0-9]{4}/,
-    );
+    expect(renderedWrapper[0].children[0].data).toMatch(regexDatetime);
+    expect(renderedWrapper[1].children[0].data).toMatch(regexUpdatedDatetime);
   });
 
-  describe('time dependent tests', () => {
-    describe('tests after 10 am', () => {
-      const inBritishSummerTime = isBritishSummerTime(Date.now());
-      const timeZoneString = inBritishSummerTime ? 'BST' : 'GMT';
-
-      beforeEach(() => {
-        // sets time to 2017-05-31T13:00:00.000Z BST
-        // or 2017-01-01T13:00:00.000Z GMT
-        // needs to be after 10am at least so the > 10 hour logic can be tested
-        const timestamp = inBritishSummerTime ? 1496235600000 : 1483275600000;
-        Date.now = jest.fn(() => timestamp);
-      });
-
-      it('should render one absolute timestamp (with datetime) when published > 10 hours ago && today', () => {
-        const twentyThreeHoursAgo = timestampGenerator({
-          hours: 10,
-          seconds: 25,
-        });
-        const renderedWrapper = renderedTimestamps(
-          <Timestamp
-            firstPublished={twentyThreeHoursAgo}
-            lastPublished={twentyThreeHoursAgo}
-          />,
-        );
-
-        expect(renderedWrapper.length).toEqual(1);
-        expect(renderedWrapper[0].children[0].data).toMatch(regexDatetime);
-      });
-
-      it('should render absolute time (with datetime) for lastPublished and for firstPublished if published today > 10 hrs ago', () => {
-        const firstPublishedTwelveHoursAgo = timestampGenerator({
-          hours: 12,
-        });
-        const lastPublishedElevenHoursAgo = timestampGenerator({
-          hours: 11,
-        });
-        const renderedWrapper = renderedTimestamps(
-          <Timestamp
-            firstPublished={firstPublishedTwelveHoursAgo}
-            lastPublished={lastPublishedElevenHoursAgo}
-          />,
-        );
-
-        expect(renderedWrapper.length).toEqual(2);
-        expect(renderedWrapper[0].children[0].data).toMatch(regexDatetime);
-        expect(renderedWrapper[1].children[0].data).toMatch(
-          /Updated [0-9]{1,2} \w+ [0-9]{4}[,] [0-9]{2}[:][0-9]{2} \w+/,
-        );
-        expect(renderedWrapper[1].children[0].data).toContain(timeZoneString);
-      });
+  it('should render two timestamps - published: date, updated: date when firstPublished before today and lastPublished beforeToday but not same day as firstPublished', () => {
+    const threeDaysAgo = timestampGenerator({
+      days: 3,
     });
+    const twoDaysAgo = timestampGenerator({ days: 2 });
+    const renderedWrapper = renderedTimestamps(
+      <Timestamp firstPublished={threeDaysAgo} lastPublished={twoDaysAgo} />,
+    );
+    expect(renderedWrapper.length).toEqual(2);
+    expect(renderedWrapper[0].children[0].data).toMatch(regexDate);
```

Comment:
When reading through the file, it can be difficult to glance at these assertions and know exactly what's happening.  It will aid readability and maintainability to refactor to have helper methods 

```
    const firstChild = wrapper => wrapper[0].children[0].data;
    const secondChild = wrapper => wrapper[1].children[0].data;
```
which can then be used 
```
expect(firstChild(renderedWrapper)).toMatch(regexDate);
expect(secondChild(renderedWrapper)).toMatch(regexUpdatedDate);
```